src/App.tsx

```diff
@@ -13,39 +27,97 @@
   'Honey',
   'Jam',
   'Garlic',
-];
+] as const;
+
+function sortItems(goods: Good[], { sortType, queue }: FilterParams) {
+  const items = [...goods];
+
+  if (sortType) {
+    items.sort((a, b) => {
+      switch (sortType) {
+        case SortType.alphabet:
+          return a.localeCompare(b);
+
+        case SortType.length:
+          return a.length - b.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (queue) {
+    items.reverse();
+  }
+
+  return items;
+}
 
 export const App: React.FC = () => {
+  const [queueField, setQueueField] = useState(false); // Булеве значення
+  const [sortField, setSortField] = useState<SortType>(SortType.default);
+  const visibleGoods = sortItems([...goodsFromServer], {
+    sortType: sortField,
+    queue: queueField,
+  });
+
+  const isResetVisible = sortField || queueField;
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        {/* Кнопка сортування за алфавітом */}
+        <button
+          type="button"
+          className={classnames('button', 'is-info', {
+            'is-light': sortField !== SortType.alphabet,
```

Comment:
The checklist suggests marking the active button to indicate to the user which sorting option is currently active. Consider adding a visual indicator for the active sorting button.